### PR TITLE
Adding a api.all.<status>.count and measure

### PR DESCRIPTION
**Pull request by jonasfj**
I didn't run tests... Just quickly hacked it online, but I think this is what I meant...

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ export function expressMiddleware (monitor, name) {
           monitor.measure(k, d[0] * 1000 + d[1] / 1000000);
           monitor.count(k);
         }
+        monitor.measure(['all', stat], d[0] * 1000 + d[1] / 1000000);
```
Comment (imbstack) [inline src/utils.js]: I think you want to either move this inside the loop and end up with 
`taskcluster-queue.api.all.client-error.5m.count` and `taskcluster-queue.api.all.all.5m.count` or replace `stat` in these new lines with `success` and only end up with `taskcluster-queue.api.all.client-error.5m.count`.

I might be misunderstanding though.

Comment (jonasfj) [inline src/utils.js]: Ah, that might be right...
Actually, I'm little suspicious as to the utility of all these alls.. some of them sure...

But given that 99% is success, then we might not need both all and success. As the percentiles will be the same...

Comment (jonasfj) [inline src/utils.js]: Ah, now I see the bug, thanks...

**Comment by imbstack** [comment]
## r+

I'll merge this now.

**Pull request merged**